tests/lorax/test_lorax_bootiso.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+#####
+#
+# Builds a boot.iso with lorax and boots it with QEMU
+#
+#####
+
+set -e
+
+. /usr/share/beakerlib/beakerlib.sh
+. "$(dirname $0)/lib/lib.sh"
+
+CLI="${CLI:-./src/sbin/lorax}"
+
+
+# Make up a name (slightly unsafe), should not exist before running lorax so use -u
+OUTPUT_DIR=$(mktemp -d -u '/var/tmp/lorax-output.XXXXX')
+rlJournalStart
+    rlPhaseStartSetup
+        rlAssertExists "$QEMU_BIN"
```

Comment:
This test is redundant. We know that qemu is installed on the test machines. Also, you'd get an error when trying to execute it later anyway.

Reply:
I think atodorov added this because of the qemu name differences in distributions. Better to find out before you waste the time building the iso only to have to boot fail. I don't think it hurts anything to keep this.

Reply:
It doesn't hurt indeed, but we will waste the time anyway, because beakerlib doesn't stop a test when something fails.

Reply:
still it says FAIL when you read the test protocol (the output from beakerlib) and that is far more easy to grasp instead of looking for a "command not found" in the raw output.

Reply:
It will - of course - also say FAIL when the command cannot be found.